src/App.js

```diff
@@ -28,14 +28,16 @@ const App = () => {
       <div className="App">
         <Switch>
           <Route exact path="/">
+            <Home />
```

Comment:
If you store the token in state at App level, you can use `<Redirect>` for each path:
```jsx
{userToken ? <Redirect to="/list" /> : <Home />}
```